unit-tests/algo/thermal-loop/create-syntetic-data.h

```diff
@@ -0,0 +1,36 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+//#cmake:add-file ../../../src/algo/thermal-loop/*.cpp
+#include "../algo-common.h"
+#include "algo/thermal-loop/l500-thermal-loop.h"
+
+using namespace librealsense::algo::thermal_loop::l500;
```

Comment:
If this file is only for l500, l500 should be in its name?

Reply:
changed to "create-synthetic-l500-thermal-table.h"